index.js

```diff
@@ -1,71 +1,120 @@
 const { Octokit } = require('@octokit/rest');
 const q = require('daskeyboard-applet');
-const dotenv = require('dotenv');
-
-dotenv.config();
 
 const colors = {
-    pending: '#FFA500',
-    success: '#00FF00',
-    failure: '#FF0000',
+  pending: '#FFA500',
+  success: '#00FF00',
+  failure: '#FF0000',
 };
 
 const logger = q.logger;
 
+class Status {
+  static PENDING = 'pending';
+  static SUCCESS = 'success';
+  static FAILURE = 'failure';
+}
+
 class GitHubActions extends q.DesktopApp {
-    constructor() {
-        super();
-        this.pollingInterval = 60000;
-        this.octokit = new Octokit({ auth: process.env.GITHUB_TOKEN });
-        logger.info('GitHub Actions Tracker ready to go!');
-    }
-    async run() {
-        const actions = await this.getGitHubActions();
-        const points = this.generatePoints(actions);
+  constructor() {
+    super();
+    this.pollingInterval = 60000;
+    this.octokit = new Octokit({ auth: this.authorization.apiKey });
+    this.owner = this.options.owner;
+    this.repo = this.options.repo;
+    logger.info('GitHub Actions Tracker ready to go!');
+  }
+
+  async run() {
+    const actions = await this.getGitHubActions();
```

Comment:
You should not try to do the call if the config is not set yet otherwise it will error